### ISSUE TITLE
Export/import 

### ISSUE THREAD

**Issue by Hadatko** [andrey-pr/OpenBafangTool#16]
It would be nice to have ability to export bike details into file (json?). And possible import (can work as backup and restore, but there are also useful data like model number,....)

**Comment by andrey-pr** [comment]
Yes, its possible. Maybe I'll add this feature, but currently I have no free time. Also, backups are already available, but currently without auto restore (only as a last resort and for restoring "by hands"). Every time when program reads bike configuration, it saves backup file on disk (the file is not send anywhere by network and is stored only locally)

**Comment by andrey-pr** [comment]
Do you need export only for backup or you have some another idea how to use it?

**Comment by Hadatko** [comment]
Actually i didn't want to use it as back up. I wanted download bike details like model in user friendly way and quickly

**Comment by andrey-pr** [comment]
> Actually i didn't want to use it as back up. I wanted download bike details like model in user friendly way and quickly

What is this feature should be usable for? Do you want to print all data on paper automatically or use export as API for your separate software or etc?

**Comment by Hadatko** [comment]
I wanted just store data in computer and later maybe use e.g. model for searching details on internet. I am programmer so i was thinking fo some exporting button and export data....

**Comment by andrey-pr** [comment]
> I wanted just store data in computer and later maybe use e.g. model for searching details on internet. I am programmer so i was thinking fo some exporting button and export data....

I want to know use cases of this function to know how to implement it. For example, if you need export to open manualy or print, it should export in pdf, if you want to open exported data with custom python script, it should be in json, etc
